packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1273,6 +1274,17 @@ module.exports = function<T, P, I, TI, PI, C, CC, CX, PL>(
   // TODO: Everything below this is written as if it has been lifted to the
   // renderers. I'll do this in a follow-up.
 
+  // Ensure performAsyncWork gets wrapped. This currently needs to be lazy because
+  // we use dynamic injection. TODO: Make this eagerly wrapping this callback once
+  // we have static injection.
+  let hasCallbackBeenScheduled: boolean = false;
+  function ensureCallbackWrapped() {
+    if (!hasCallbackBeenScheduled) {
+      performAsyncWork = wrapEventListener('reactAsyncWork', performAsyncWork);
```

Comment:
This is pretty unfortunate but we have to make these lazy. I also don't want to keep wrapping each individual call to rIC because this wrapper is pretty heavy.